### DISCUSSION TITLE
Group crawling / scraping

### DISCUSSION THREAD

**Discussion by BodomBeach** [TLYu0419/facebook_crawler#9] [General]
Hi, I saw you tried to scrape groups by POSTing on graphql API, is it working ? 
Can you get the next POST requests after that ?

**Comment by tlyu0419** [comment]
Hi @BodomBeach 

Thank you for your question. 
It's a known problem about the group crawler function. 
I should have more time to fix it next month, and I will notify you when I finish it.
Thanks!
